### ISSUE TITLE
Clicking the icon in bottom bar does not open ad4m

### ISSUE THREAD

**Issue by leifriksheim** [coasys/ad4m#263]
Clicking on the icon in the app dock on macOS does not trigger visibility of the launcher. 

Also what about removing splash screen and fullscreen mode and just do everything from the tray so when you are done creating an agent the ui doesnt dissapear.

**Comment by fayeed** [comment]
Yeah, I guess we can remove it, but the idea was that it centre and front of the app so users would know that there was an error if anything broke.

How about this if the user is opening the launcher for the first time we can show the modal so they can read up on the ad4m and create the user but on the next run we don't and open it from the tray but if there's an error we show the modal.

what do you think?

**Comment by leifriksheim** [comment]
The only thing that I'm worried about is when you create an agent and it is successful it will close the window, and you won't know that it is actually working and you would need to click the tray icon again. Could we get around this somehow?

**Comment by fayeed** [comment]
I was thinking we could maybe show a popup in the system tray which says "Ad4m is minimized, click here to open it" or something similar which goes away after 10 seconds or so.
